### ISSUE TITLE
Guidance (or defining an extra table) for providing measurements

### ISSUE THREAD

**Issue by peterdesmet** [tdwg/camtrap-dp#358]
I've heard of a number of use cases that want to provide more detailed **measurements**, such as:

- Deployment level: Deployment failure duration, deployment failure type (@ddashs #354, @CrocutaLupus #355)
- Media level: temperature recorded by device (@dhobern)
- Observation level: fish length (@sachitrajbhandari)

There are two ways to do this:

1. Provide this information in [**deploymentTags**](https://camtrap-dp.tdwg.org/data/#deployments.deploymentTags), [**observationTags**](https://camtrap-dp.tdwg.org/data/#observations.observationTags) as pipe-delimited key:value pairs. Note there is no `mediaTags` and that it might be hard to parse that information.
2. Provide this information in a **custom table** (Camtrap DP allows this, see e.g. this [embedded json table](https://github.com/tdwg/camtrap-dp/blob/ebf585d9cb1d033878ae8611e15b845a7c48fea3/example/datapackage.json#L30-L40)). Such custom table would have their own Table Schema to define the fields and a related identifier to the deployment, media, or observation.

The second option is better from a machine-readability perspective, since the data can be provided in a neatly tabular format. The drawback of a _custom_ table is that measurements provided in one Camtrap DP dataset are likely going not going to be interoperable with measurements from another dataset. It might therefore be good to add a 4th table for measurements. We could adopt the [OBIS Extended Measurement Or Fact extension](https://rs.gbif.org/extension/obis/extended_measurement_or_fact_2023-08-28.xml). It structures measurements as rows, not as columns, meaning that Camtrap DP doesn't have to define the types of measurements, just the structure to provide measurements.

In any case, I think it would be useful to provide guidance on how to provide measurements in the FAQ, as it is a common use case.

**Comment by dhobern** [comment]
Thanks @peterdesmet - this is great. Simple guidance or pointers should help many people.

**Comment by kbubnicki** [comment]
A generic solution for recording different types of measurements, as suggested above by @peterdesmet,  would be really cool I think!

The FAQ (#329)  is critically needed for a wide adoption of Camtrap DP. It would be great to have somebody willing to contribute and help with this!

@dhobern: here is a small example of how one can use the custom-table-approach (based on some measurements by [PhenoCams](https://phenocam.nau.edu/webcam/)): https://gist.github.com/kbubnicki/4cbb41e3ecc7c5c4290417bf91e88d92

**Comment by ben-norton** [comment]
@kbubnicki You can put some structure around the measurements table based on the type of measurement.
See https://bioportal.bioontology.org/ontologies/OBOE/?p=summary
